Gopkg.toml

```diff
@@ -55,7 +55,7 @@ required = [
   name = "github.com/giantswarm/errors"
 
 [[constraint]]
-  branch = "master"
+  branch = "kubernetes-1.15.4"
```

Comment:
To be reverted once https://github.com/giantswarm/k8sportforward/pull/19 gets merged.